Automattic/republication-tracker-tool
# Optional attribution statement; invisible pixel

### Changes proposed in this Pull Request:



Closes #100.
Closes #69.

### How to test the changes in this Pull Request:

1. Ensure you have the plugin widget inserted somewhere (e.g. sidebar), so it's usable
2. Use the plugin - visit a post, use the widget to copy content, republish somewhere
3. Observe that the republished content has an attribution statement at the end, followed by the site icon (the one that's used for the favicon - you can set it in Customizer->Site Identity)
4. Observe that the republished content contains a 1 by 1 pixel tracking image
5. Visit the plugin settings, which are at the bottom of "Reading" settings in WP Admin Settings
6. Observe that "Display attribution" setting is on by default. Turn it off
7. Redo step 2 and observe that the copied content does not contain the attribution statement and site icon, but it does contain the tracking pixel image
